src/common/components/contact-input-field/ContactInputField.tsx

```diff
@@ -0,0 +1,60 @@
+import { ExclamationCircleIcon } from '@heroicons/react/solid';
+import React from 'react';
+import { classNames } from '../../helpers/Tailwind.helper';
+
+const ContactInputField = (props: {
+  config: Partial<any>;
```

Comment:
poate in loc de any ar trebui sa avem o interfata

Reply:
am pus interfata